pages/_app.js

```diff
@@ -182,9 +182,9 @@ MyApp.getInitialProps = async (appContext) => {
     "/hubs/[slug]",
     "/hubs/[slug]/live",
     "/paper/[paperId]/[paperName]",
+    "/doc-v2/[documentType]/[documentId]/[documentSlug]",
```

Comment:
`doc-v2` is a temporary route which will be removed once we are ready to make the switch